contracts/samples/callback/TokenCallbackHandler.sol

```diff
@@ -4,25 +4,14 @@ pragma solidity ^0.8.12;
 /* solhint-disable no-empty-blocks */
```

Comment:
I think that you can remove this line because is no longer needed. What do you think? :)

Reply:
Makes sense, have pushed a fix